go.mod

```diff
@@ -21,6 +21,61 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require (
```

Comment:
I have no clue why, but `go get` adds this whole `require` block and I dunno how to remove it.

Reply:
Try to remove the entire block and run `go mod tidy` instead. That should give a better result.

Reply:
same identical outcome

Reply:
I tried locally - it seems to be the `golang` version? I reverted as my change failed in CI.

Reply:
I'm on 1.17